site/encrypt.md

```diff
@@ -1,6 +1,6 @@
 ## The Encrypt Model Tool
 
-**NOTE: The current encryption algorithms require JDK 8 to execute, in order to meet Oracle's security standards.  While it is possible to run WLST with a newer JDK than what was used to install WebLogic Server, WLST on older versions of WebLogic Server 10.3.x may not work properly with JDK 8, out of the box, due to JDK 6 JVM arguments that have been removed in JDK8.  It may be necessary to modify the WLST start scripts to remove JVM arguments that have been removed between JDK 6 and JDK 8.**
+**NOTE: The current encryption algorithms require JDK 8 to execute, in order to meet Oracle's security standards.  While it is possible to run WLST with a newer JDK than what was used to install WebLogic Server, WLST on the version of WebLogic Server 10.3.6 may not work properly with JDK 8, out of the box, due to JDK 6 JVM arguments that have been removed in JDK8.  It may be necessary to modify the WLST start scripts to remove JVM arguments that have been removed between JDK 6 and JDK 8.**
```

Comment:
The current encryption algorithms require JDK 8 to execute, in order to meet Oracle's security standards. -> To meet Oracle's security standards, the current encryption algorithms require JDK 8.
WLST on the version of WebLogic Server 10.3.6 may not work properly -> WLST in WebLogic Server 10.3.6 may not work properly
It may be necessary to modify the WLST start scripts to remove JVM arguments that have been removed between JDK 6 and JDK 8. -> You may need to modify WLST start scripts to remove the JDK 6 JVM arguments that have been removed in JDK 8.